src/nbsap/views/indicators.py

```diff
@@ -23,3 +23,15 @@ def indicator(request, pk):
     return render(request, 'indicators/indicator_details.html', {
         'indicator': indicator,
     })
+
+
+def list_indicators(request, pk=None):
+    if not pk:
+        indicator = models.EuIndicator.objects.all()[0]
```

Comment:
We should use first(), but that's django 1.6 and we're stuck with 1.5 here, right?